pyproject.toml

```diff
@@ -16,6 +16,8 @@ dependencies = [
     "ruff>=0.9.9",
     "tokenizers>=0.21.0",
     "tqdm>=4.67.1",
+    "beautifulsoup4,
+    "urllib3,
```

Comment:
Oh, but this is adding the dependencies to the main pyproject.toml file in the root directory. I thought you were gonna add it inline on the script, based on what we talked :-p. Or either add your own pyproject file in the dof_word folder :-p